policyengine_us/variables/gov/states/mo/tax/income/taxable_income/mo_net_state_income_taxes.py

```diff
@@ -24,22 +24,29 @@ def formula(tax_unit, period, parameters):
             "state_and_local_sales_or_income_tax", period
         )
 
-        # Technically, state and local income tax could include the "local_earnings_tax" variable,
-        # but because the definition of net_state_income_taxes is simply (state_tax + local_tax) - local_tax,
+        # Technically, state and local income tax could include the
+        # "local_earnings_tax" variable,
+        # but because the definition of net_state_income_taxes is
+        # simply (state_tax + local_tax) - local_tax,
         # it is redundant to keep it
-        # More info about local income tax from the Federal W2 on page 26 here: https://dor.mo.gov/forms/MO-1040%20Instructions_2021.pdf
+        # More info about local income tax from the Federal W2 on page 26 here:
+        # https://dor.mo.gov/forms/MO-1040%20Instructions_2021.pdf
 
         net_state_income_taxes = tax_unit("state_income_tax", period)
 
```

Comment:
```suggestion
        # Compute the share of SALT from net state income taxes.
        # Use a mask rather than where to avoid a divide-by-zero warning. Default to one.
```